### PR TITLE
Bump to convex 1.3.1 and add format param to api calls

### PR DESCRIPTION
**Pull request by nipunn1313**
Got `go test` to pass!
Uses the unreleased record type, but 🤷

### CODE REVIEW

File: convex_test.go [hunk 1]
```diff
@@ -13,12 +13,12 @@ import (
 )
 
 func clear(c *ConvexDB) {
-	c.mutation(&UdfExecution{Path: "clear", Args: []interface{}{}})
+	c.mutation(&UdfExecution{Path: "clear", Args: map[string]interface{}{}, Format: "json"})
 }
 
 // Test saving and loading links for SQLiteDB
 func Test_Convex_SaveLoadLinks(t *testing.T) {
-	db := NewConvexDB("https://feeble-gull-946.convex.cloud", "test")
```
Comment (gautamg795) [inline convex_test.go:21]: should these changes be committed? not sure if it really matters

Comment (nipunn1313) [reply convex_test.go:21]: Yeah - this is just my dev deployment instead of yours
Let me see if I can have it read from .env.local

Comment (nipunn1313) [reply convex_test.go:21]: got it to work!

**Comment by gautamg795** [comment]
looks good; the build on main is broken right now because an upstream dep was yanked so I'm going to fix that, rebase your PR, then merge

**Comment by gautamg795** [comment]
merging proved difficult; I'm just cherry picking the commit to fix the build

**Review by gautamg795 — APPROVED** [review]

**Pull request merged**